Update recruiting.md (#476)
mattermost/mattermost-handbook: contributors/join-us/recruiting.md

## Interview Stages

> As a fully remote and global team, Mattermost interviewers are located in different timezones. Since we also interview candidates across different time zones, we schedule interviews one-by-one vs. as a panel of interviews on one specific date. Below are the typical stages and amount of interviewers a candidate can expect in a full interview process at Mattermost. 

Reviewer: ```suggestion
As a fully remote and global team, Mattermost interviewers are located in different time zones. Since we also interview candidates across different time zones, we schedule interviews one-on-one as opposed to a panel of interviews on one specific date. Below are the typical stages and number of interviewers a candidate can expect in a full interview process at Mattermost.
```

> All interviews will take place over video. Interviewers will wait 5 minutes for a candidate to join before leaving the interview. We encourage candidates to email their recruiting contact and the Mattermost interviewer if they are not able to make a previously scheduled interview. 

Reviewer: ```suggestion
All interviews take place over video (using Zoom). Interviewers will wait 5 minutes for a candidate to join before leaving the interview. We encourage candidates to email their recruiting contact and the Mattermost interviewer if they are not able to make a scheduled interview.
```

> 1. **Recruiter Screen:** This person is a candidate's main point of contact during the interview process at Mattermost. For some roles, Hiring Managers may incorporate a recruiter screen within their first interview with a candidate.

Reviewer: ```suggestion
1. **Recruiter Screen:** This is a candidate's main point of contact during the interview process at Mattermost. For some roles, Hiring Managers may incorporate a recruiter screen within their first interview with a candidate.
```
